composer.json

```
@@ -5,7 +5,8 @@
         "php": "^7.0.0",
         "illuminate/http": "^5.1",
         "illuminate/database": "^5.1",
-        "langleyfoxall/helpers-laravel": "^1.10"
+        "langleyfoxall/helpers-laravel": "^1.10",
+        "illuminate/support": "^5.8"
```

I believe 5.1 supports `Str::snake`.

```suggestion
        "illuminate/support": "^5.1"
```